common/config/rush/nonbrowser-approved-packages.json

```diff
@@ -93,6 +93,10 @@
       "name": "chai",
       "allowedCategories": [ "libraries", "other" ]
     },
+    {
+      "name": "chalk",
+      "allowedCategories": [ "libraries" ]
+    },
```

Comment:
We use colors, not chalk.

Reply:
we use chalk in GCB apparently, however it was not defined as a dependency. PNPM exposed this as a problem. I can replace chalk with colors in this PR, or I can also do it in another PR. your call

---
In reply to: [159794773](https://github.com/Microsoft/web-build-tools/pull/383#discussion_r159794773) [](ancestors = 159794773)

Reply:
File an issue, use a separate PR

---
In reply to: [159795444](https://github.com/Microsoft/web-build-tools/pull/383#discussion_r159795444) [](ancestors = 159795444,159794773)